app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -10,12 +10,24 @@ class ManageIQ::Providers::Autosde::StorageManager::CloudVolume < ::CloudVolume
 
   def self.raw_create_volume(ext_management_system, options = {})
     # @type [StorageService]
-    vol_to_create = ext_management_system.autosde_client.VolumeCreate(
-      :service => ext_management_system.storage_services.find(options["storage_service_id"]).ems_ref,
+    creation_hash = {
+      :service => "",
       :name    => options["name"],
       :size    => options["size"],
       :count   => options["count"]
-    )
+    }
+
+    if options['mode'] == 'Basic'
```

Comment:
Where is `options['mode']` coming from?  I don't see it in the DDF params

Reply:
it used to be in `cloud-volume-form.schema.js` but I moved it to `cloud_voume.rb` following @Fryguy's request: 
https://github.com/ManageIQ/manageiq-ui-classic/pull/8582#issuecomment-1379580040